woocommerce/woocommerce-gutenberg-products-block
# Product specific select screen improvements.

Fixes #32 
Fixes #30 
Closes #29  

To test: 
1. Update to latest version of Gutenberg if needed.
2. Verify grid is the default layout when inserting a new block.

All of the following steps are for "Individual products mode":

3. Add products and verify they give visual feedback when added (they fade out and stuff)
4. Verify the search dropdown closes when you click outside of the search area.
